pkg/manifests/manifests_test.go

```diff
@@ -1428,46 +1428,6 @@ func TestPrometheusK8sControlPlaneRulesFiltered(t *testing.T) {
 	}
 }
 
-func TestPrometheusEtcdRulesFiltered(t *testing.T) {
```

Comment:
No clue what this is, I think its legacy?

Reply:
Haha, not completely until the etcd service monitor moves to CEO too :)
But yes it's fine to remove the tests.

https://github.com/openshift/cluster-monitoring-operator/blob/8ec92e1c442dcca2a8d73e10303fd37543745835/pkg/operator/operator.go#L641-L647